nf_core/lint/params_used.py

```diff
@@ -28,6 +28,8 @@ def params_used(self):
         if cf in ignore_params:
             ignored.append("Config variable ignored: {}".format(self._wrap_quotes(cf)))
             continue
+        if cf.count(".") > 1:
+            continue
```

Comment:
Might be nice to add a `log.debug` here, if nothing else just as a sort of code comment 😅

Reply:
Yep agree, I thought about it then didn't do it because it wouldn't look nice :eye: 

Think I'll add both!